particle-iot/device-os
# Rename SoM platform names [ch32184]

### Problem

The Device OS `PLATFORM_NAME` definition for platforms 22, 23, 24 started as `argon-som`, `boron-som` and `xenon-som` in 1.1.0-rc.1 and 1.1.0-rc.2. The official platform names for the SoMs are A SoM, B SoM and X SoM so in Device OS the `PLATFORM_NAME` should be renamed to `asom`, `bsom`, `xsom` for compilation.

### Solution

- Update build scripts to reflect the name changes
- Update USB device descriptor info
- Update comments and PLATFORM_ARGON_SOM throughout the code to PLATFORM_ASOM, etc..
- Update previous 1.1.0-rc.x release notes.

### Steps to Test

- If this build compiles, that will test Travis CI
- Local compilation should also pass:
```
time ./make_release.sh --platform=asom --tests --debug --publish=1.1.0-rc.2
time ./make_release.sh --platform=bsom --tests --debug  --publish=1.1.0-rc.2
time ./make_release.sh --platform=xsom --tests --debug  --publish=1.1.0-rc.2
```

---

### Completeness


---

- [internal] Rename SoM platform names [ch32184] [#1774](https://github.com/particle-iot/device-os/pull/1774)
